Makefile

```diff
@@ -295,7 +295,7 @@ all: buildenv-dirs
 
 .PHONY: all-local
 # Run tests, cleanup dependencies, and generate CRDs locally
-all-local: test deps configsync-crds
+all-local: test deps clientgen configsync-crds
```

Comment:
`clientgen` or `clientgen-in-docker`? Same for `configsync-crds`?

Reply:
`clientgen`. `all-local` is wrapped by `test-presubmit` to run all of these in a docker container.